cves/CVE-2018-11784.yml

```diff
@@ -44,15 +44,19 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description:
```

Comment:
Could you maybe put this in your own words? This is super similar to the website's description so I don't fully understand it.